services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -562,7 +562,8 @@ protected void initDataProvider() {
             .blockManager(blockManager)
             .attestationManager(attestationManager)
             .isLivenessTrackingEnabled(
-                beaconConfig.beaconRestApiConfig().isBeaconLivenessTrackingEnabled())
+                beaconConfig.beaconRestApiConfig().isBeaconLivenessTrackingEnabled()
```

Comment:
I will just separate this in a separate method: `getLivenessTrackingEnabled()` and pass the `beaconConfig`